src/auditwheel/main_repair.py

```diff
@@ -85,6 +85,12 @@ def configure_parser(sub_parsers):
         help="Strip symbols in the resulting wheel",
         default=False,
     )
+    p.add_argument(
+        "--exlude",
```

Comment:
```suggestion
        "--exclude",
```

Reply:
this is great work @martinRenou ! we need this in hpc environment to exclude libraries like OpenMP/OpenACC.

As Alex suggested, naming option `exclude` is more obvious here? (or, this was just a typo)

Reply:
This was a typo indeed, thanks for reviewing!